constructor/header.sh

```diff
@@ -517,13 +519,13 @@ cat <<EOF
 __CONCLUSION_TEXT__
 EOF
 
-if [ "$PYTHONPATH" != "" ]; then
+if [ "${PYTHONPATH:-}" != "" ]; then
     printf "WARNING:\\n"
     printf "    You currently have a PYTHONPATH environment variable set. This may cause\\n"
     printf "    unexpected behavior when running the Python interpreter in __NAME__.\\n"
     printf "    For best results, please verify that your PYTHONPATH only points to\\n"
     printf "    directories of packages that are compatible with the Python interpreter\\n"
-    printf "    in __NAME__: $PREFIX\\n"
+    printf "    in __NAME__: %s\\n" "$PREFIX"
```

Comment:
How come we escape the `\n` newline here but not in the other instance I mentioned above?

Reply:
that escaping is not required, but both leads to the same output see e.g. https://unix.stackexchange.com/questions/564998/why-doesnt-printf-escape-newlines

Reply:
Got it.

Seems like single quotes are usually preferred for the `printf` format string, but we can do that in a separate PR.